Localization.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Localization.h"
+
+void Localization::loadTranslations(const std::string& language,
+    const std::unordered_map<std::string, std::string>& translations) {
+    translations_[language] = translations;
+}
+
+void Localization::setLanguage(const std::string& language) {
+    if (translations_.find(language) != translations_.end()) {
+        current_language_ = language;
+    }
+}
+
+std::string Localization::getLocalizedMessage(const std::string& key) const {
+    auto lang_itr = translations_.find(current_language_);

```

Comment:
Any reason to use `find`, instead of directly indexing using the `current_language_`?